yt/frontends/boxlib/data_structures.py

```diff
@@ -571,7 +571,13 @@ def _read_particles(self, directory_name, is_checkpoint, extra_field_names=None)
             self.ds._particle_type_counts = {}
         self.ds._particle_type_counts[directory_name] = num_parts
 
-        base_particle_fn = self.ds.output_dir + '/' + directory_name + "/Level_%d/DATA_%.4d"
+        base = self.ds.output_dir + '/' + directory_name
+        if (len(glob.glob(base + "/Level_?/DATA_????")) > 0):
```

Comment:
It's maybe wiser to use `os.path.join(base, 'Level_?', 'DATA_????'` instead, as it would support other platforms thant *nix.